### DISCUSSION TITLE
2014/01/its-the-backblaze-restores-that-fail/

### DISCUSSION THREAD

**Discussion by SeanKilleen** [SeanKilleen/seankilleen.github.io#622] [Specific Post Comments]
https://seankilleen.com/2014/01/its-the-backblaze-restores-that-fail/

**Comment by SeanKilleen** [comment]
## Historical Comments

2017-06-21, Disqus User [hfxmike](https://disqus.com/by/hfxmike/): 

> What did you end up doing? I'm trying my first restore with BackBlaze now - 3 GB. It's downloading very slow, and who knows if it will unzip.


2017-06-21, Disqus User [hfxmike](https://disqus.com/by/hfxmike/): 

> It unzipped, but 2 of the files were missing - the local client app said that everything was backed up....I guess that $5/mo for unlimited backup was too good to be true. Download speed averaged 700KB/s! blech

2017-06-21, Me:

> I ended up eventually ordering the external drive from Backblaze. A few hundred dollars later, I had the data back. Switched to CrashPlan -- had a great experience so far, including a restore.
> 
> Lots more options, plus for a little bit more I got 10 licenses, which I gave to my family, etc. CrashPlan provides a local backup option, so pieces of my backups live on the computers of those I know. If anything truly critical goes down, I can get it from them a lot faster.

2017-08-22, DIsqus User [Morgan Feldon](https://disqus.com/by/morganfeldon/):

> And now that Crashplan has gotten rid of it's inexpensive plans, this topic is popping up again. Backblaze is not reputable. Crashplan and Amazon are now pricing themselves out of the market for end users. What's the solution?

2018-02-24, Disqus user [foxyshadis](https://disqus.com/by/foxyshadis/):

> Local backup, one of the things I miss about Mozy. (Their pricing was so obnoxious though.) If Backblaze would allow simultaneous local/online backup it would be just about perfect.

2018-02-24, Me: 

> Have to say, I agree. I've been searching for a solid piece of local backup software, and considering doing something with that and using it to back up to Wasabe for storage (like S3, but appears to have the edge a bit)

**Comment by apkuki** [comment]
Hey - i used https://erickutcher.github.io/ and that was a charm...

With Chrome Inspect you can get the download Link URL in the Script behind the the WebUI Download Button

I had to get rid of 2MFA and enter Username and PW in HTTP Downloader but now i can actually get a properly manageable  queue ... at least of these 5 items i can restore with private account

Also Speed is waaaay better!
